### PR TITLE
GUI_ChangeInstrumentsPage

### PR DESCRIPTION
**Pull request by dTarasova**

### CODE REVIEW

**Review by Cerberous007 — COMMENTED** [review]
image
I can go to page that does not exist, and it`s hard to go back without refreshing

**Review by Cerberous007 — COMMENTED** [review]
image
What about making button of the current page smaller?

**Review by nmamro — COMMENTED** [review]
Strange hat with "go to" we can go to like 999999 page and that we can edit text in box showing current page number, also "go to" disappears at last page

**Review by Cerberous007 — APPROVED** [review]

File: Clients/GUI/Pages/Instruments.razor [hunk 1]
```diff
@@ -60,33 +60,74 @@
...
+
+                            @if (pageIndex + 2 < lastPageIndex)
+                            {
+                                <input class="btn btn-light btn-sm col-sm-1" readonly="readonly" value="..." @onclick="@(() => MoveTo(pageIndex + 3))" />
+
+                                <input class=" btn btn-light btn-sm col-sm-1" readonly="readonly" value="@lastPageIndex" @onclick="@(() => MoveTo(lastPageIndex))" />
+                            }
+                        }
+                        <input type=" @(nextDisabled ? "hidden" : "button" )"
+                               class="btn btn-primary" value="Next" @onclick="@NextPage" />
```
Comment (ifelseelif) [inline Clients/GUI/Pages/Instruments.razor:97]: Add attribute readonly

**Review by nmamro — APPROVED** [review]

**Review by denis-kriv — APPROVED** [review]

**Review by ifelseelif — APPROVED** [review]

**Pull request merged**